src/controllers/message.controller.js

```diff
@@ -0,0 +1,45 @@
+import { Message } from '../models/Message.model.js';
+import { EventEmitter } from 'events';
+
+const MESSAGE = 'message';
+
+export const emitter = new EventEmitter();
+
+export const getMessages = async (req, res) => {
+  const whereClauses = {};
+  const { roomId } = req.query;
+
+  whereClauses.roomId = roomId;
```

Comment:
This line assumes that 'roomId' is always provided in the query parameters. Consider adding a check to ensure 'roomId' is present before assigning it to 'whereClauses.roomId'.